devfiles/hosted-che-dogfooding.devfile.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# See: https://sipb.mit.edu/doc/safe-shell/
+#
+#
+# A devfile to develpop che-theia projects on https://che.openshift.io.
+# For developers who work on Theia core or che-theia core extensions and plugins.
```

Comment:
What is `che-theia core`?
> For developers who work on Theia core

Why only Theia core https://github.com/eclipse-theia/theia/tree/master/packages/core
What about other Theia packages?

Reply:
Thanks for the feedback. It really looks strange. I just copied the description from the che-theia-all.devfile.yaml